src/app/_components/attachment/attachment.component.html

```diff
@@ -4,7 +4,7 @@
       </span>
     <div class="psdk-modal-file-selector" *ngIf="bShowSelector$">
         <mat-spinner *ngIf="bLoading$" class="progress-spinner" diameter="40"></mat-spinner>
-        <input hidden type="file" #uploader (change)="uploadMyFiles($event)"/>
+        <input hidden type="file" #uploader id='upload-photo' (change)="uploadMyFiles($event)"/>
```

Comment:
Since this can be used for uploads in general, perhaps we should change the name to something more generic such as upload-input?

Reply:
sure John, I will modify and update PR